test/e2e/engine-ollama-pvc/test.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+source $REPO_DIR/test/e2e/common.sh
+
+models_release="kubeai-models"
+
+PV_HOST_PATH=/usr/share/ollama/.ollama/models/
```

Comment:
```
+ PV_HOST_PATH=/usr/share/ollama/.ollama/models/                                                                         
+ mkdir -p /usr/share/ollama/.ollama/models/                                                                             
mkdir: cannot create directory ‘/usr/share/ollama’: Permission denied 
```
we should use a directory that generally works locally e.g.: `PV_HOST_PATH=/tmp/model`